aten/src/ATen/BatchingRegistrations.cpp

```diff
@@ -413,6 +420,20 @@ TORCH_LIBRARY_IMPL(aten, Batched, m) {
   BINARY_POINTWISE_VA(rsub, Scalar);
   BINARY_POINTWISE(mul);
   BINARY_POINTWISE(div);
+
+  // at::pow has three overloads
```

Comment:
There's more than three:
```
  nt_impl(m, "pow.Tensor_Tensor_out", NestedTensor_pow_out_1);
  nt_impl(m, "pow.Tensor_Tensor", NestedTensor_binary<at::pow>);
  nt_impl(m, "pow_.Tensor", NestedTensor_pow__1);
  nt_impl(m, "pow.Tensor_Scalar_out", NestedTensor_pow_out_2);
  nt_impl(m, "pow.Tensor_Scalar", NestedTensor_pow_2);
  nt_impl(m, "pow.Scalar_out", NestedTensor_pow_out_3);
  nt_impl(m, "pow.Scalar", NestedTensor_pow_3);
```

Reply:
I'm not handling the in-place or the out variants yet, just the out-of-place variant for now

Reply:
I can update the comment to say that there are "three out-of-place overloads". Did you have any other comments?